src/quoridor/server/server.go

```diff
@@ -66,14 +70,27 @@ func getGameHandler(w http.ResponseWriter, r *http.Request) {
 	response.SendOK(w, game)
 }
 
+func joinGameHandler(w http.ResponseWriter, r *http.Request) {
+	id := request.GetGameID(r)
+	gameToken:= shortuuid.New()
+	game, err := gamecontroller.JoinGame(id, gameToken)
+	if err != nil {
+		response.SendBadRequestError(w, err)
+		return
+	}
+	w.Header().Set("Quoridor-Token", gameToken)
+	response.SendOK(w, game)
```

Comment:
You should return the token as json in the response, not the header.

Header is used to send "meta" values to the server for each action, not to return data to the user